src/settings/SettingsModal.tsx

```diff
@@ -62,37 +62,46 @@ const SettingsModal = ({
         <ModalHeader>Settings</ModalHeader>
         <ModalCloseButton />
         <ModalBody>
-          <VStack>
+          <VStack align="flex-start">
             <RenderIf shouldRender={!hideArButton}>
               <FormControl display="flex" alignItems="center">
                 <FormLabel htmlFor="ar-toggle" mb="0">
-                  Enable Augmented Reality?
+                  Enable Augmented Reality
                 </FormLabel>
                 <Switch
                   id="ar-toggle"
                   isChecked={arEnabled}
-                  onChange={onChangeArToggle}
+                  onChange={(e) => {
+                    onChangeArToggle(e.target.checked);
+                  }}
+                  style={{ marginLeft: 'auto' }}
```

Comment:
Can we set this with a Chakra prop rather than inline style?

Reply:
Fixed!